### PR TITLE
fix assign grammar of application agent

### PR DESCRIPTION
**Pull request by jidalong**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Since variable "defaultAgent" was assigned by ":=" in line 273, so we should delete  the declared line 268.
**Does this PR introduce a user-facing change?**:
NONE

```release-note

```

### CODE REVIEW

File: cloud/pkg/dynamiccontroller/application/application.go [hunk 1]
```diff
@@ -270,7 +270,7 @@ var once sync.Once
 
 // edged config.Config.HostnameOverride
 func NewApplicationAgent() *Agent {
-	defaultAgent := &Agent{nodeName: metaserverconfig.Config.NodeName}
+	defaultAgent = &Agent{nodeName: metaserverconfig.Config.NodeName}
```
[the remark this answers is not in the document]
Comment (jidalong) [reply cloud/pkg/dynamiccontroller/application/application.go]: Okay~ I have delete the global variable instead.

**Comment by zc2638** [comment]
```golang
var once sync.Once
```
According to the `NewApplicationAgent ` method, we probably shouldn't use `once` to execute.

**Comment by jidalong** [comment]
> ```go
> var once sync.Once
> ```
> 
> According to the `NewApplicationAgent ` method, we probably shouldn't use `once` to execute.

@zc2638 Should we just change the part of `NewApplicationAgent ` method 
> ```go
>once.Do(func() {
>	go wait.Until(func() {
>		defaultAgent.GC()
>	}, time.Minute*5, beehiveContext.Done())
>})
> ```
to the flowing code, and delete the declare "var once sync.Once".
> ```go
>go wait.Until(func() {
>     defaultAgent.GC()
>}, time.Minute*5, beehiveContext.Done())
> ```

**Comment by zc2638** [comment]
@jidalong 
I think so.

**Comment by jidalong** [comment]
> @jidalong I think so.

Okey, I think it would be better as well~and has change it.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**